scripts/src/metrics/metrics.py

```diff
@@ -57,7 +58,29 @@ def send_release_metrics(write_key, downloads):
 
     for provider in metrics:
         for chart in metrics[provider]:
-            send_metric(write_key,provider,f"{chart} downloads", metrics[provider][chart])
+            number_of_downloads=0
+            for key,value in metrics[provider][chart].items():
+                chart_downloads.append({"downloads":number_of_downloads,"name":key,"provider":provider})
+    chart_downloads.sort(key = lambda i: i[0],reverse=True)
+
+    for x in range(len(chart_downloads)):
+        send_download_metric(write_key,chart_downloads[x]["provider"],chart_downloads[x]["downloads"],chart_downloads[x]["name"],x+1)
+
+    for x in range(5):
```

Comment:
One thing here we don't want the more than one version of any chart in the top 5. Also only the latest version of the chart should be considered. So if a chart has version 1.0.0 at 500 downloads and version 1.1.0 at 200 downloads we  would only consider 1.1.0 with 200 downloads for the top 5.

Reply:
Only for the top 5 , different versions should not be considered right, but they still need to written insegment with the different id